modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -88,11 +88,14 @@ def get_claims(veis_token, btsss_token)
       end
 
       symbolized_body = response.body.deep_symbolize_keys
-      parse_claim_date = ->(c) { Date.parse(c[:modified_on]) }
-      symbolized_body[:data].sort_by(&parse_claim_date).reverse!
+      parse_claim_date = ->(c) { Date.parse(c[:modifiedOn]) }
+      symbolized_body[:data].sort_by!(&parse_claim_date).reverse!
```

Comment:
I purposefully didn't use the `!` version of `sort_by` so we don't mess with the original array. What did you notice that was wrong with this approach?

Reply:
I realized the reason the API response was missing the top-level `data` property was due to this function. Before, when this was the last line:
```ruby
symbolized_body[:data].sort_by(&parse_claim_date).reverse!
```
the method was returning just the claims array, since that's what's getting sorted. Now the whole thing gets returned as `symbolized_body`. My thought was the mutation is fine since it's an upstream request that isn't getting persisted, so it doesn't modify anything outside the scope of this method. Would it be better to do something along the lines of returning this?
```ruby
{ :data => symbolized_body[:data].sort_by(&parse_claim_date).reverse! }
```

Reply:
I see, thanks for clarifying. I would prefer that second approach. I know it probably doesn't matter in this case, but I've been bitten so many times before by modifying data in place. 

Let me know if you're against this, happy to work it out.